src/lib/provable/foreign-field.ts

```diff
@@ -101,26 +100,33 @@ class ForeignField {
    *   class SmallField extends createForeignField(17n) {}
    *   class LargerField extends createForeignField(19n) {}
    * 
-   *   let smallField = new SmallField(10);
-   *   let largerField = new LargerField(17);
+   *   let smallField: SmallField = ...;
+   *   let largerField: LargerField = ...;
    *   Gadgets.ForeignField.assertLessThan(largerField.value, SmallField.modulus);
    *   let x = new SmallField(largerField);
```

Comment:
this whole example doesn't work anymore with the new error. you can remove it, it's really not a common use